src/status_im2/contexts/chat/bottom_sheet_composer/view.cljs

```diff
@@ -0,0 +1,137 @@
+(ns status-im2.contexts.chat.bottom-sheet-composer.view
+  (:require
+    [quo2.foundations.colors :as colors]
+    [react-native.core :as rn]
+    [react-native.gesture :as gesture]
+    [react-native.hooks :as hooks]
+    [react-native.reanimated :as reanimated]
+    [reagent.core :as reagent]
+    [utils.i18n :as i18n]
+    [status-im2.contexts.chat.bottom-sheet-composer.style :as style]
+    [status-im2.contexts.chat.bottom-sheet-composer.images.view :as images]
+    [utils.re-frame :as rf]
+    [status-im2.contexts.chat.bottom-sheet-composer.utils :as utils]
+    [status-im2.contexts.chat.bottom-sheet-composer.constants :as c]
+    [status-im2.contexts.chat.bottom-sheet-composer.actions.view :as actions]
+    [status-im2.contexts.chat.bottom-sheet-composer.keyboard :as kb]
+    [status-im2.contexts.chat.bottom-sheet-composer.sub-view :as sub-view]
+    [status-im2.contexts.chat.bottom-sheet-composer.effects :as effects]
+    [status-im2.contexts.chat.bottom-sheet-composer.gesture :as drag-gesture]
+    [status-im2.contexts.chat.bottom-sheet-composer.handlers :as handle]))
+
+(defn sheet
+  [insets window-height blur-height opacity background-y]
+  [:f>
```

Comment:
Sorry, another comment about the use of functional components in this namespace :sweat_smile: